specs/provider.yaml

```diff
@@ -913,6 +913,43 @@ paths:
           description: |
             If the request could not be processed successfully, a useful error
             message should be displayed to the user.
+
+  /resources/{id}/measures:
+    get:
+      summary: Get how much a resource has used its features
+      description: |
+        Manifold will call this endpoint daily to get usage information about a
+        resource and its features.
+
+        The Provider should only need to hold information about the time left
+        in the current month and the previous month.
```

Comment:
Add something similar to this which defines how the billing months are defined:
`Months are defined by the Manifold billing period which starts at the first of each month in UTC-0 and ends at the start of the next.`